region/maridia/outer/Mt. Everest.json

```diff
@@ -989,12 +996,17 @@
       "name": "Suitless Frozen Crab",
       "requires": [
         "canSuitlessMaridia",
-        "canTrickyUseFrozenEnemies"
+        "canTrickyUseFrozenEnemies",
+        {"or": [
+          "canTrickyJump",
+          "canSpringBallJumpMidAir",
```

Comment:
Looking back through these today, I think all of these Everest examples are good - not sure why I was confused before. Anyways, this spring ball jump feels harder than the tricky spin jump option to me. It's pretty hard to do when buffering the jump through the pause, and the crab positioning is pretty precise.
```suggestion
          "canTrickySpringBallJump",
```